### ISSUE TITLE
Error On View Preauthorizations, Load Failed & Permission Denied

### ISSUE THREAD

**Issue by wbraswell** [RapidApp/tpf-blog-live#8]
While logged in as my now-normal "wbraswell" username, I tried to view my 2 preauthorizations, which gives an error:
![perlfoundation_20191005-rapiblog_preauth_error](https://user-images.githubusercontent.com/1772630/66258770-cb048280-e76e-11e9-8f78-f438aef5a802.png)

This does not appear to be related to my experiments in changing usernames, because no matter what username I use I still get the same "Permission Denied" error.

**Comment by vanstyn** [comment]
Right, well, we've now limited the preauthorizations to admins (thats also why you no longer see it in the nav tree). This was one of the other concerns which was brought up, that normal users were given access to too much stuff, so I locked it down.

You really don't need to see the preauthorizations anyway. What those are is the guts of the system that drives things like the e-mail login links. The concept being you are "pre-authorized" to login using the access token (i.e. the link in the e-mail), once. So its really just debug info.

How did you open that page to begin with? Was it a url you saved from before, or did you make a saved search? Just curios.

Separately, thank you for what you said in the mail thread. I'm more than happy to do the same for blogs.perl.org if i'm asked... but I'm not interested in stepping on anyone's toes. I avoid politics and stick to technology ;)
